pkg/sdn/plugin/ovscontroller.go

```diff
@@ -182,18 +185,18 @@ func (oc *ovsController) ensureOvsPort(hostVeth string) (int, error) {
 	return oc.ovs.AddPort(hostVeth, -1)
 }
 
-func (oc *ovsController) setupPodFlows(ofport int, podIP, podMAC string, vnid uint32) error {
+func (oc *ovsController) setupPodFlows(ofport int, podIP, podMAC, note string, vnid uint32) error {
 	otx := oc.ovs.NewTransaction()
 
 	// ARP/IP traffic from container
-	otx.AddFlow("table=20, priority=100, in_port=%d, arp, nw_src=%s, arp_sha=%s, actions=load:%d->NXM_NX_REG0[], goto_table:21", ofport, podIP, podMAC, vnid)
-	otx.AddFlow("table=20, priority=100, in_port=%d, ip, nw_src=%s, actions=load:%d->NXM_NX_REG0[], goto_table:21", ofport, podIP, vnid)
+	otx.AddFlow("table=20, priority=100, in_port=%d, arp, nw_src=%s, arp_sha=%s, actions=load:%d->NXM_NX_REG0[], note:%s, goto_table:21", ofport, podIP, podMAC, vnid, note)
+	otx.AddFlow("table=20, priority=100, in_port=%d, ip, nw_src=%s, actions=load:%d->NXM_NX_REG0[], note:%s, goto_table:21", ofport, podIP, vnid, note)
```

Comment:
You only need to add the note to the table=20 arp flow, not every flow for this pod. (Even if we needed info from some other flow later you could still get the ContainerID->port mapping from the first flow and then recognize the later flow by the port number.)